pkg/cmd/openshift-kube-apiserver/cmd.go

```diff
@@ -0,0 +1,93 @@
+package openshift_kube_apiserver
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/coreos/go-systemd/daemon"
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	configapilatest "github.com/openshift/origin/pkg/cmd/server/apis/config/latest"
+	"github.com/openshift/origin/pkg/cmd/server/origin"
+)
+
+const RecommendedStartAPIServerName = "openshift-kube-apiserver"
+
+type OpenShiftKubeAPIServerServer struct {
+	ConfigFile string
+	Output     io.Writer
+}
+
+var longDescription = templates.LongDesc(`
+	Start an apiserver that contains the kube-apiserver resources`)
```

Comment:
sounds confusing. "Start the OpenShift kube-apiserver"

Reply:
which is the short description. "Start the extended kube-apiserver with OpenShift security extensions".

Reply:
sure